Cartfile.resolved

```diff
@@ -2,3 +2,4 @@ github "cezheng/Fuzi" "3.1.2"
 github "dexman/Minizip" "1.4.0"
 github "edrlab/GCDWebServer" "3.6.2"
 github "krzyzanowskim/CryptoSwift" "1.3.1"
+github "readium/r2-shared-swift" "ef446a7ab96910ab78381841465c5d104310b87e"
```

Comment:
this should point to our repo, not readium's

Reply:
sure, updated